src/utils/code.ts

```diff
@@ -103,10 +103,10 @@ const buildStyledProps = (propsNames: string[], childComponent: IComponent) => {
   return propsContent
 }
 
-const returnConditionalValue = (
+const returnConditionalValue: (
   propsNames: string[],
   childComponent: IComponent,
-) => {
+) => any = (propsNames: string[], childComponent: IComponent) => {
```

Comment:
why any?

Reply:
its possible the type is a string, I'll change it to boolean | string